packages/calling/src/CallHistory/callHistoryFixtures.ts

```diff
@@ -439,3 +633,47 @@ export const ERROR_DETAILS_400 = {
   },
   message: 'FAILURE',
 };
+
+/*
+ * MOCK_LINES_API_CALL_RESPONSE simulates a successful response from the UCM lines API.
+ */
+export const MOCK_LINES_API_CALL_RESPONSE: UCMLinesResponse = {
+  statusCode: 200,
+  data: {
+    lines: {
+      devices: [
+        {
+          name: 'CSFheliosucm01',
+          model: 503,
+          lines: [
+            {
+              dnorpattern: '+14928000001',
+              index: 1,
+              label: '',
+            },
+            {
+              dnorpattern: '+14928000003',
+              index: 2,
+              label: '',
+            },
+          ],
+        },
+      ],
+    },
+  },
+  message: 'SUCCESS',
+};
+
+/**
+ * MOCK_LINES_API_CALL_RESPONSE_WITH_NO_LINEDATA simulates a successful UCM lines API response
+ * where no line data is present. The `lines` field is empty, indicating no devices or lines available.
+ */
+export const MOCK_LINES_API_CALL_RESPONSE_WITH_NO_LINEDATA: UCMLinesResponse = {
+  statusCode: 200,
+  data: {
+    lines: {
```

Comment:
The question mark in Line 161 of CallHistory.ts is not being taken care of with this change. We shouldn't have lines object at all. Is this new object for a different condition in the code?

Reply:
Handled the optional scenario for the lines object.